info.go

```diff
@@ -182,3 +207,24 @@ func scanFdInfoReader(r io.Reader, fields map[string]interface{}) error {
 
 	return nil
 }
+
+// DisableStatsFunc stops collecting statistics of eBPF programs.
+type DisableStatsFunc func() error
+
+// EnableStats starts the measuring of the runtime (in ns)
+// and run counts of eBPF programs.
+//
+// Collecting statistics can have an impact on the performance.
+//
+// Requires at least 5.8.
+func EnableStats(which uint32) (DisableStatsFunc, error) {
```

Comment:
```suggestion
func EnableStats(which uint32) (io.Closer, error) {
```

IIRC internal.FD should already implement `Closer`. That way we have one less exported type, and returning an interface is more flexible than a function. If the API sprouts some additional usage of the FD we can extend the returned interface.